experimental/terraform/setup.sh

```diff
@@ -30,14 +30,22 @@ export BILLING_ACCOUNT=$(gcloud beta billing projects describe ${PARENT_PROJECT}
 echo "Creating new project for Four Keys Dashboard..."
 gcloud projects create ${FOURKEYS_PROJECT} --folder=${PARENT_FOLDER}
 gcloud beta billing projects link ${FOURKEYS_PROJECT} --billing-account=${BILLING_ACCOUNT}
+export PARENT_PROJECTNUM=$(gcloud projects describe ${PARENT_PROJECT} --format='value(projectNumber)')
 
 # FOR DEVELOPMENT ONLY: purge all TF state
+echo "Purging TF state [FOR DEVELOPMENT ONLY]"
 rm -rf .terraform terraform.tfstate* terraform.tfvars
 
+# build the event-handler container (using parent project) and stash it in the fourkeys project
+echo "building event-handler container"
+gcloud services enable cloudbuild.googleapis.com
+gcloud services enable containerregistry.googleapis.com --project=${FOURKEYS_PROJECT}
+gcloud projects add-iam-policy-binding ${FOURKEYS_PROJECT} --member="serviceAccount:${PARENT_PROJECTNUM}@cloudbuild.gserviceaccount.com" --role="roles/storage.admin"
```

Comment:
what's this for?

Reply:
Same as above -- to push a container from one project into another project, I need this permission. Thanks for the review!